config/schemapatch/kueue-operator.crd.yaml

```diff
@@ -193,6 +225,60 @@ spec:
                                   additionalProperties:
                                     type: string
                               x-kubernetes-map-type: atomic
+                    manageJobsWithoutQueueName:
+                      description: |-
+                        ManageJobsWithoutQueueName controls whether or not Kueue reconciles
+                        jobs that don't set the annotation kueue.x-k8s.io/queue-name.
+                        If set to true, then those jobs will be suspended and never started unless
+                        they are assigned a queue and eventually admitted. This also applies to
+                        jobs created before starting the kueue controller.
+                        Defaults to false; therefore, those jobs are not managed and if they are created
+                        unsuspended, they will start immediately.
+                      type: boolean
```

Comment:
probably should be a enum/string, booleans aren't considered good practice anymore

Reply:
Yea, but the config option is a bool. So if I add an enum I can convert it to a bool for the kueue config that kueue reads. But that seems a bit silly.

Reply:
I agree its silly but I think the point still stands

Reply:
so I will create a enum of True and False and convert that to a bool?

Reply:
Alright I went ahead and updated this.